### PR TITLE
[OBSDEF-6934] update iam and fed deployments with resource definitions

### PR DESCRIPTION
**Pull request by bst525**
## Purpose
[OBSDEF-6934](https://jira.cec.lab.emc.com:8443/browse/OBSDEF-6934)
added resources to iam and fed pods

## PR checklist
- [ ] make test passed
- [x] make build passed
- [x] helm install <chart> passed
- [ ] vSphere7 make package deployments passed
- [ ] helm upgrade <chart> passed
- [ ] helm test <release_name> passed

## Testing
charts-custom-ci   has passed

https://asd-ecs-jenkins.isus.emc.com/jenkins/job/charts-custom-ci/119/

### CODE REVIEW

**Review by eddiepavkovic — APPROVED** [review]
ok

File: objectscale-iam/templates/deployment.yaml [hunk 1]
```diff
@@ -21,6 +21,11 @@ spec:
       {{- end }}
       containers:
       - name: iam
+        resources:
+          limits:
+            memory: 2Gi
+          requests:
+            memory: 2Gi
```
Comment (hldnova) [inline objectscale-iam/templates/deployment.yaml:28]: should be the request be smaller than the limit? It seems that the configuration for other pods in ecs cluster is about a 3rd to half of the limits.

File: federation/templates/federation_deployment.yaml [hunk 2]
```diff
@@ -22,6 +22,11 @@ spec:
       {{- end }}
       containers:
       - name: fedsvc
+        resources:
+          limits:
+            memory: 500M
+          requests:
+            memory: 500M
```
Comment (hldnova) [inline federation/templates/federation_deployment.yaml:29]: same as in iam, should this be little lower than the limit?

File: objectscale-iam/templates/deployment.yaml (hunk 1 above)
Comment (bst525) [reply objectscale-iam/templates/deployment.yaml:28]: Peter and I have found they need to be equal.

**Review by SeemaTahaliyani — APPROVED** [review]

File: objectscale-iam/templates/deployment.yaml (hunk 1 above)
Comment (hldnova) [reply objectscale-iam/templates/deployment.yaml:28]: hmm. i am curious why it would be the case.

**Review by eddiepavkovic — APPROVED** [review]
lgmt

File: objectscale-iam/templates/deployment.yaml (hunk 1 above)
Comment (bst525) [reply objectscale-iam/templates/deployment.yaml:28]: https://eos2git.cec.lab.emc.com/ECS/ecs-flex-operator/pull/331/files

**Pull request merged**